js/src/modal/modal.spec.js

```diff
@@ -38,6 +222,16 @@ describe('Modal', () => {
     })
 
     document.body.style.paddingRight = '0px'
+
```

Comment:
You should remove the modal CSS added and when you made changes on `document.body` style

Reply:
The modal css is being removed below on line 233

Reply:
Body style is reset on line 216

Reply:
![image](https://user-images.githubusercontent.com/1689750/64097064-8480ca00-cd63-11e9-81a3-83577673617b.png)

I don't see anything which clears the body style 😟

Reply:
Ok I saw it thanks 👍